example/program.yml

```diff
@@ -0,0 +1,9 @@
+- title: Opening Remarks
+  start-time: 2020-11-16T10:45:00
+  end-time: 2020-11-16T11:00:00
+- title: Zoom Q&A Session 1i: Sentiment Analysis, Stylistic Analysis, and Argument Mining
+  start-time: 2020-11-16T12:00:00
+  end-time: 2020-11-16T13:00:00
+  papers:
+    - 1
```

Comment:
Refer to the papers by ID. This makes the interface format less redundant, but also makes them harder to manually spot-check. What do you think is a better user experience?